examples/platform/efr32/efr32_creds.h

```diff
@@ -0,0 +1,57 @@
+#ifndef MATTER_DEVICE_CREDENTIALS_EFR32
+#define MATTER_DEVICE_CREDENTIALS_EFR32
+
```

Comment:
Please add a comment that this is just boilerplate for the examples and it doesn't need to be cut'n'pasted, and where actual CD for a product should be set if you set `MATTER_DEVICE_CREDENTIALS_EFR32`